EIPS/eip-7830.md

```diff
@@ -60,16 +58,12 @@ A simple analysis shows contract deployements for 64 KiB contracts to be between
 
 Note that the Max 30M gas contract size of 135 KiB is outside the limits of what is proposed in this EIP and is included to show what it would take to exceed current gas limits.
 
-<!-- https://docs.google.com/spreadsheets/d/1C2dd5sVnZNKXOpRknHhxt6MnTTN50c3b9d6ZU2rvqDQ/edit?usp=sharing -->
```

Comment:
@shemnon this is not accessible. If you want to make it public, can you post it on the magicians thread?

Reply:
Made viewable, but being in a comment it will be deleted when going to final/last call.

Reply:
Hence why I asked to post it on the magicians thread if it is still relevant.

Reply:
It's posted to the thread, you can delete.